### PR TITLE
fixed broken loop

### PR DESCRIPTION
**Pull request by privatezero**
Eliminated package path being set outside the loop for checksum package.

### CODE REVIEW

**Comment by dericed** [comment]
Can you test this with referencing to packages via full path and with relative path.
```
checksumpackage /Users/private_zero/important/EXAMPLE_AIP
```
and
```
cd /Users/private_zero/important/
checksumpackage EXAMPLE_AIP
```

**Comment by privatezero** [comment]
https://github.com/mediamicroservices/mm/pull/84/commits/d119b2942b08596486914bb8359f6e944982112b should do it!

File: checksumpackage [hunk 1]
```diff
@@ -114,6 +106,8 @@ _version_dfxml(){
 }
 
 while [ "${*}" != "" ] ; do
+    ABS_PATH=$(cd "$1"; pwd)
```
Comment (dericed) [inline checksumpackage]: Should add a check somewhere that "$1" is a directory. `if [ ! -d "$1" ] ; then echo OH NOOOOO ; fi`

File: checksumpackage [hunk 2]
```diff
@@ -114,6 +106,8 @@ _version_dfxml(){
...
+    INPUT=$(basename "$1")
     _log -b
     PACKAGE="${ABS_PATH}"
     if [ -d "${PACKAGE}" ] ; then
```
Comment (privatezero) [inline checksumpackage]: Would the directory validation be redundant with this check?

Comment (privatezero) [inline checksumpackage]: On examination, that check was indeed insufficient!!  Fixed with https://github.com/mediamicroservices/mm/pull/84/commits/2e422689e0d262196adac488f2add13ce0d09d00 - script should now break loop and exit on illegal input.

**Pull request merged**